AWSutils.sh

```diff
@@ -350,6 +350,10 @@ function InstallCfnBootstrap {
       chroot "${CHROOTMNT}" ln -sf /opt/aws/apitools/cfn-init/init/redhat/cfn-hup /etc/init.d/cfn-hup || \
          err_exit "Failed creating symlink for cfn-hup service"
 
+      err_exit "Making sure cfn-hup service is executable..." NONE
+      chroot "${CHROOTMNT}" chmod +x /etc/init.d/cfn-hup || \
```

Comment:
Marginally less expensive to just do:
~~~
      chmod +x "${CHROOTMNT}/etc/init.d/cfn-hup" || \
~~~

Reply:
ok done!